wand/tools.py

```diff
@@ -28,7 +28,19 @@ class LockException(Exception):
 
 def get_config_path(args, name_suffix=""):
     config_file = "{}{}_config.pyon".format(args.name, name_suffix)
-    wand_dir = os.path.dirname(wand.__file__)
+
+    home = str(Path.home())
```

Comment:
This could be `QStandardPaths::writeableLocation(QStandardPaths::AppLocalDataLocation)`.

Reply:
(As in, the whole data_dir computation.)

Reply:
@dnadlinger thanks, I wasn't aware of that so always good to learn new API.

I'm currently out of time for further work on this so won't have time to integrate and test that change -- particularly given that the current solution works fine. Please feel free to make the change thought if it bugs you.